prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -73,6 +75,33 @@ object ConstantSubstitutor {
     }
 }
 
+/**
+ * Custom FHIR commands created by report stream to help map from FHIR -> HL7
+ * only used in cases when the same logic couldn't be accomplished using the FHIRPath
+ */
+object CustomFHIRCommands {
```

Comment:
Let's call these functions instead of commands, to match the FHIR Path spec terms.